### ISSUE TITLE
[🐛] not working with yarn

### ISSUE THREAD

**Issue by Spown** [snyk/snyk#1374]
- `node -v`: v10.22.0
- `npm -v`: 6.14.6 (not used anywhere, see yarn)
- `yarn -v`: 1.22.5
- `snyk -v`: 1.385.2
- OS: Windows 10x64
- Command run: snyk protect && snyk monitor

Probably related to #667, only I use yarn

When run via ``yarn run`` I get:
```
> Executing task: yarn run snyk-protect <

yarn run v1.22.5
$ snyk protect && snyk monitor
Error: Cannot find module 'graphlib'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (-----------\node_modules\@snyk\snyk-cocoapods-plugin\node_modules\@snyk\dep-graph\src\core\builder.ts:1:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Steps to reproduce
1. have in your package.json
```json
  "scripts": {
    "snyk-protect": "snyk protect && snyk monitor",
```
2. ``> yarn run snyk-protect``

thank you.

**Comment by JackuB** [comment]
Hey, couldn't replicate it locally with similar setup, with fresh install.

Do you see `graphlib` in output of `yarn list`? Or the `@snyk/graphlib`? Snyk CLI is currently using a fork of `graphlib` called `@snyk/graphlib` - do you see either of those in the yarn.lock?

**Comment by Spown** [comment]
yes ``"@snyk/graphlib@2.1.9-patch"`` and ``"@snyk/graphlib@2.1.9-patch.2"`` are present in the `` yarn.lock``. ``node_modules/@snyk/graphlib`` is also present on disk.

**Comment by JackuB** [comment]
Hi, we worked on a PR #1388 that will remove those `patch` versions from the dependency tree. We hope it'll resolve this issue. Will update here once the PR is merged.

**Comment by JackuB** [comment]
Hey, the https://github.com/snyk/snyk/releases/tag/v1.393.1 was released, could you please try updating your Snyk CLI to the latest version? `@snyk/depgraph` is not part of the dependency tree anymore, so you shouldn't see it in the lockfile after upgrading

<img width="517" alt="Screenshot 2020-09-14 at 13 42 52" src="https://user-images.githubusercontent.com/1788727/93081852-31d53000-f690-11ea-9ed0-19d240979a3b.png">

**Comment by Spown** [comment]
@JackuB seems to work fine, thank you!
